compiler/luci/import/src/ImporterEx.test.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2025 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/ImporterEx.h"
+
+#include <gtest/gtest.h>
+#include <sstream>
+
+TEST(ImporterEx, uses_default_error_handler_NEG)
+{
+  std::ostringstream cerr_substitute;
+
+  auto cerr_original_buffer = std::cerr.rdbuf();
+  std::cerr.rdbuf(cerr_substitute.rdbuf());
+
+  luci::ImporterEx importer;
+  importer.importVerifyModule("/non/existent.path");
+
+  std::cerr.rdbuf(cerr_original_buffer);
+
+  ASSERT_GT(cerr_substitute.str().length(), 0);
+}
+
+TEST(ImporterEx, calls_external_error_handler_NEG)
+{
+
```

Comment:
```suggestion
```

Reply:
This is an empty line added by the formatter. If I remove it the code formatting CI check fails.

Reply:
this is strange. @hseok-oh , is this intended?

I assume this empty line comes from following `struct ErrorHandler`
can you please move `ErrorHandler` outside of this function ?

Reply:
Ok, I've re-ran the formatter after removing this line and this time it left it as is. I'm almost certain this line was added by the formatter previously though.

In this case I'd rather leave this struct inside the test because it provides scoping and the struct is intended to be used only in this particular test.

Reply:
ok thanks for double checking